catalog/spatial/registry/registry-api-impl/pom.xml

```diff
@@ -241,22 +241,22 @@
                                         <limit>
                                             <counter>INSTRUCTION</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.75</minimum>
+                                            <minimum>0.50</minimum>
```

Comment:
I was hoping to grab that ticket myself and fix this there, but I will add tests improve the coverage in this PR.